fury/ui/core.py

```diff
@@ -685,6 +685,10 @@ class TextBlock2D(UI):
         Adds text shadow.
     size : (int, int)
         Size (width, height) in pixels of the text bounding box.
+    auto_font_scale : bool
+        Automatically scale font according to the text bounding box.
+    dynamic_bbox : bool
+        Automatically resize the bounding box according to the content.
```

Comment:
if you add them here, I agree with @tvcastillod, we need setter/getter functions.

Reply:
I will remove this for now and will create a subsequent PR for adding these attributes.